### PR TITLE
RHV-16816-lp Landing page for Installing OCP RHV

### PR DESCRIPTION
**Pull request by rolfedh**
This is a temporary landing page with a link to the "Quickstart Guide: Installing OpenShift on Red Hat Virtualization" article on https://access.redhat.com/articles/4903411. More formal doc coming soon.

### CODE REVIEW

**Comment by rolfedh** [comment]
Applies to 4.4 GA

**Comment by rolfedh** [comment]
@openshift/team-documentation

File: installing/installing_rhv/installing-rhv.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+= Installing a cluster on Red Hat Virtualization
+include::modules/common-attributes.adoc[]
+:context: installing-rhv
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster
+in a Red Hat Virtualization (RHV) environment.
+
+The installer that deploys OCP on RHV automates the process using what’s called “installer-provisioned infrastructure.” This installer is available for Linux and macOS only.
```
Comment (jboxman) [inline installing/installing_rhv/installing-rhv.adoc]: s/OCP/{product-title}/

I'm not sure I'd do

> using what’s called “installer-provisioned infrastructure.”

maybe

_automates the process by using installer-provisioned infrastructure._

File: installing/installing_rhv/installing-rhv.adoc [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+This link:https://access.redhat.com/articles/4903411[Quickstart Guide: Installing OpenShift on Red Hat Virtualization] shows you how to get up and running with OCP on RHV.
```
Comment (jboxman) [inline installing/installing_rhv/installing-rhv.adoc]: _To learn more about installing a {product-title} cluster on RHV, read link:https://access.redhat.com/articles/4903411[Quickstart Guide: Installing OpenShift on Red Hat Virtualization]_

File: modules/installation-overview.adoc [hunk 3]
```diff
@@ -71,6 +57,7 @@ installer-provisioned infrastructure on the following platforms:
 * Google Cloud Platform (GCP)
 * Microsoft Azure
 * Red Hat OpenStack Platform version 13 and 14
+* Red Hat Virtualization
```
Comment (jboxman) [inline modules/installation-overview.adoc]: Could probably add acronym: (RHV)

**Review by jboxman — CHANGES_REQUESTED** [review]
@rolfedh, added feedback.

File: installing/installing_rhv/installing-rhv.adoc (hunk 2 above)
Comment (rolfedh) [reply installing/installing_rhv/installing-rhv.adoc]: In the link text, I replaced "OpenShift" with {product-title}, okay?

**Comment by jboxman** [comment]
@rolfedh, do you want me to merge this and CP to 4.4?

Thanks!

**Comment by rolfedh** [comment]
> @rolfedh, do you want me to merge this and CP to 4.4? 
Yes. Please do. 
I'm also checking with Peter L. whether to CP it to 4.3 as well.

**Comment by rolfedh** [comment]
Yes. Please also CP to 4.3.

**Comment by rolfedh** [comment]
Just CP to 4.4. Not 4.3. Thanks!

**Review by jboxman — APPROVED** [review]

**Pull request merged**